WikiWatershed/model-my-watershed
# Updates to Splash and Header text

## Overview

Makes small changes to the text and links in the splash an header bar in MMW mode.

Connects #2623 

### Notes

The main implementation detail noted in conversation is the unusual import used in 2ac1f9f, which allows a cheap escape hatch from the initialization call stack which would otherwise prevent an `App` instance to be available.  I tried various refactors, but I think a comprehensive solution is outside this scope and perhaps outside the remaining project budget.

Additionally:
* The word "Monitor" was added to Step 2, but the description was not updated as per the issue.  The language as it exists may not be sufficient to explain the Monitor concept @ajrobbins 
* Previously, in BigCZ mode, one could get to My Projects which would then awkwardly convert the app to MMW mode without updating any title elements.  I decided this was a bug and it is no longer possible to get to the Project listing in BigCZ mode from the user or header links.

## Testing Instructions

A new image has been added, use `./scripts/bundle.sh --vendor` in addition to your normal bundling workflow.

MMW:
* Ensure the text & images appear as described on the splash screen and header
* While logged out, use the Projects header link to be prompted then redirected to the Project listing
* While logged in, do the same and do not get prompted
* There is no longer a Project link in the user drop down

BigCZ:
* Ensure the splash & header text remains unchanged
* The user drop down nor the header include a link to My Projects
